### PR TITLE
Restore DefaultNamespaceDeletionTimeout workaround

### PR DESCRIPTION
**Pull request by mangelajo**
DefaultNamespaceDeletionTimeout was set to 10 minutes at some
point because of kubernetes#47135.

Another option would be just removing the TODO.

Signed-off-by: Miguel Angel Ajo <majopela@redhat.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by mangelajo** [comment]
CLA signed

**Comment by mangelajo** [comment]
/assign @fabriziopandini
---
irc: ajo / mangelajo
Miguel Angel Ajo Pelayo
+34 636 52 25 69
skype: ajoajoajo

**Review by timothysc — APPROVED** [review]
/ok-to-test
/assign @timothysc 
/approve 

hold till tests are green

**Comment by mangelajo** [comment]
I have updated the patch based on the gofmt complaints here: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/78255/pull-kubernetes-verify/1145778118884069377

**Comment by Pothulapati** [comment]
Hello! I'm part of the bug triage team for the 1.16 release cycle and considering this PR has not been updated for a long time, I'd like to check what's the status of this PR. The code freeze is starting 29th August (about 2.5 weeks from now) and while there is plenty of time, we want to ensure that each PR has a chance to be merged on time. 

As the PR is tagged for 1.16, is it still planned for this release?

**Review by alejandrox1 — CHANGES_REQUESTED** [review]
@mangelajo thank you for working on this!
right now ci is failing due to unformatted code. could you please run `./hack/update-gofmt.sh` ?

File: test/e2e/framework/framework.go [hunk 1]
```diff
@@ -54,11 +54,9 @@ import (
...
 
 const (
-	maxKubectlExecRetries = 5
-	// DefaultNamespaceDeletionTimeout is timeout duration for waiting for a namespace deletion.
-	// TODO(mikedanese): reset this to 5 minutes once #47135 is resolved.
-	// ref https://github.com/kubernetes/kubernetes/issues/47135
-	DefaultNamespaceDeletionTimeout = 10 * time.Minute
+	maxKubectlExecRetries           = 5
+    // DefaultNamespaceDeletionTimeout is timeout duration for waiting for a namespace deletion.
+	DefaultNamespaceDeletionTimeout = 5 * time.Minute
```
Comment (liggitt) [inline test/e2e/framework/framework.go]: @smarterclayton do we still need this? I thought you removed the namespace deletion wait altogether

Comment (alejandrox1) [reply test/e2e/framework/framework.go]: that's https://github.com/kubernetes/kubernetes/pull/82439 , right?

**Comment by timothysc** [comment]
I'm going to close this per - #82439 , If you feel there is still an issue please reopen and describe the issue in detail.

**Pull request closed without merging**